src/app/car.ts

```diff
@@ -0,0 +1,10 @@
+export class Car {
+
+    id: string
+    brand: string
+    registration: Date
+    country: string
+    created_at: Date
```

Comment:
why are created_at and last_updated part of the frontend model?

Reply:
Fixed.